rapids/rapids.sh

```diff
@@ -8,16 +8,16 @@ function get_metadata_attribute() {
   /usr/share/google/get_metadata_value "attributes/${attribute_name}" || echo -n "${default_value}"
 }
 
-readonly DEFAULT_DASK_RAPIDS_VERSION="22.04"
+readonly DEFAULT_DASK_RAPIDS_VERSION="22.08"
```

Comment:
@nvliyuan  this has to set it back to 22.04. This is only used for Dask.

Reply:
done